frontend/src/views/project.js

```diff
@@ -152,14 +153,37 @@ export const MoreFilters = () => {
   const navigate = useNavigate();
   const [fullProjectsQuery] = useExploreProjectsQueryParams();
 
+  useEffect(() => {
+    // Disable scrolling outside the component when it appears
+    document.body.style.overflow = 'hidden';
+    document.documentElement.style.overflow = 'hidden';
+
+    return () => {
+      // Restore scrolling when the component is closed
+      document.body.style.overflow = 'auto';
+      document.documentElement.style.overflow = 'auto';
+    };
+  }, []);
+
   const currentUrl = `/explore${
     stringify(fullProjectsQuery) ? ['?', stringify(fullProjectsQuery)].join('') : ''
   }`;
 
   return (
     <>
       <div className="absolute left-0 z-4 mt1 w-40-l w-100 h-100 bg-white h4 ph1 ph5-l">
-        <MoreFiltersForm currentUrl={currentUrl} />
+        <div
+          className="scrollable-container"
+          style={{
+            height: '100%',
+            overflow: 'auto',
```

Comment:
We use utility classes from [tachyons](https://tachyons.io/), so I recommend you use corresponding utility classes from tachyons if available. These two could be just replaced with `h-100 overflow-hidden`.

Reply:
Yes exactly! I will fix it.